cardano-testnet/test/cardano-testnet-test/files/golden/queries/refScriptSizeOut.json

```diff
@@ -0,0 +1,3 @@
+{
+    "refInputScriptSize": 8
```

Comment:
Bytes?

Reply:
I think it is the number of 2-byte words (7.5 plus padding), because the CBOR of the script is 30 hex digits, and a byte is represented with 2 hex digits. But I didn't write this, so I don't really know